### PR TITLE
webrtc: add test for canTrickleIceCandidates

### PR DESCRIPTION
**Pull request by fippo**

### CODE REVIEW

File: webrtc/rtcpeerconnection/canTrickleIceCandidates.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      assert_unreached('Error ' + e.name + ': ' + e.message);
+    }));
+  }, 'canTrickleIceCandidates property is true after setRemoteDescription with a=ice-options:trickle');
+
+  async_test(function(test) {
+    var pc = new RTCPeerConnection();
+
+    pc.setRemoteDescription(new RTCSessionDescription({type: 'offer', sdp: sdp.replace('a=ice-options:trickle\r\n', '')}))
+    .then(function() {
+      assert_equals(pc.canTrickleIceCandidates, false, 'canTrickleIceCandidates property is true after setRemoteDescription');
```
Comment (alvestrand) [inline webrtc/rtcpeerconnection/canTrickleIceCandidates.html]: s/true/false in message (I think it's supposed to describe the successful condition, not the failure mode; if i'm wrong switch it in the other place).

File: webrtc/rtcpeerconnection/canTrickleIceCandidates.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      'a=msid:stream1 track1\r\n' +
+      'a=ssrc:1001 cname:some\r\n' + 
+      'a=rtpmap:111 opus/48000/2\r\n';
+
+  test(function(test) {
+    var pc = new RTCPeerConnection();
+    assert_equals(pc.canTrickleIceCandidates, null, 'canTrickleIceCandidates property is null');
+  }, 'canTrickleIceCandidates property is null prior to setRemoteDescription');
+
+  async_test(function(test) {
```
Comment (alvestrand) [inline webrtc/rtcpeerconnection/canTrickleIceCandidates.html]: why not promise_test?

File: webrtc/rtcpeerconnection/canTrickleIceCandidates.html (hunk 1 above)
Comment (alvestrand) [inline webrtc/rtcpeerconnection/canTrickleIceCandidates.html]: why not just assert_false(pc.canTrickleIceCandidates)?

**Review by alvestrand — COMMENTED** [review]
Looks good, but
1) get the submodules to be not-included
2) some nits.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/218806989)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 4d2d27ef8d7829005268a5f0628912108aabd5b6
Using browser at version BuildID 20170404100210; SourceStamp b5d8b27a753725c1de41ffae2e338798f3b5cacd
Using browser at version BuildID 20170404100210; SourceStamp b5d8b27a753725c1de41ffae2e338798f3b5cacd
Starting 10 test iterations
Starting 10 test iterations

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/218806990)

# Chrome (unstable channel) #
Testing web-platform-tests at revision 4d2d27ef8d7829005268a5f0628912108aabd5b6
Using browser at version 59.0.3053.3 dev
Using browser at version 59.0.3053.3 dev
Starting 10 test iterations
Starting 10 test iterations

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5382/)

**Comment by fippo** [comment]
failure in Chrome is intended. Will file a bug for it.

**Review by alvestrand — APPROVED** [review]

**Pull request merged**